### ISSUE TITLE
Local docker builds fail due to unshallow

### ISSUE THREAD

**Issue by hacktobeer** [google/turbinia#750]
Google Cloud Buidl needs to unshallow a copy due to our use of setuptools_scm. If we unshallow the repo in the Dockerfile this will throw an error when building locally (as it's not a shallow clone).

```
Step 15/25 : RUN cd /tmp/ && git fetch --prune --unshallow
 ---> Running in 0e2c5ff7e33c
fatal: --unshallow on a complete repository does not make sense
The command '/bin/sh -c cd /tmp/ && git fetch --prune --unshallow' returned a non-zero code: 128
```

https://github.com/google/turbinia/blob/master/docker/server/Dockerfile#L23
* same in worker Dockerfile

**Comment by hacktobeer** [comment]
if $(git rev-parse --is-shallow-repository); then
    ... repository is shallow ...
fi

**Comment by aarontp** [comment]
Fixed by #751
